hudi-cli/src/main/java/org/apache/hudi/cli/commands/TableCommand.java

```diff
@@ -131,4 +131,14 @@ public String descTable() {
     });
     return HoodiePrintHelper.print(header, new HashMap<>(), "", false, -1, false, rows);
   }
+
+  /**
+   * Refresh table metadata.
+   */
+  @CliCommand(value = {"refresh", "refresh metadata", "commits refresh", "cleans refresh", "savepoints refresh"},
```

Comment:
`refresh metadata` to `metadata refresh`?

Reply:
> `refresh metadata` to `metadata refresh`?

Yes,  had address.